src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -565,6 +612,10 @@ ${f.resolution}
       {
         option: '-v, --toVersion [toVersion]',
         description: 'The version of SharePoint Framework to which upgrade the project'
+      },
+      {
+        option: '-tsv, --toTypeScriptVersion [toTypeScriptVersion]',
```

Comment:
Short options must consist of a single letter, so `-tsv` is in fact equal to three shorthand switches: `-t -s -v`